Add Chainstack 2.1.2 release notes retrospectively (#306)
chainstack/docs: docs/release-notes.md

# Release notes

## Chainstack 2.1.2

*November 2, 2020*

### What's new

> * **Improved billing experience**. You can now reattempt past due payments through billing card change or directly in the activity log (where you can see more information attached to your billing events). 

Reviewer: Change to **Billing**.  We put component name in bold and then expand on what's done. The component here is billing (also mentioned in previous releases)

Reviewer: Something like this:

```
You can now reattempt past due payments through billing card change or directly in the activity log. The activity log also provides more information on your billing events.
```

Please avoid brackets as much as possible.

> * **Improved billing experience**. You can now reattempt past due payments through billing card change or directly in the activity log (where you can see more information attached to your billing events). 

Author: ```suggestion
* **Billing**. You can now reattempt past due payments through billing card change or directly in the activity log. The activity log also provides more information on your billing events. 
```

### Bug fixes

> * Hyperledger Fabric Explorer being out of sync with the network state.

Reviewer 2: Perhaps we should rephrase to make it clear that only some explorers were out of sync and not all

> * Hyperledger Fabric Explorer being out of sync with the network state.

Author: ```suggestion
* Resolved an issue with some Hyperledger Fabric Explorers being out of sync with the network state.
```

> * Resolved an issue with some Hyperledger Fabric Explorers being out of sync with the network state.

Reviewer 2: BTW there's no such thing as "Hyperledger Fabric Explorer", there's "[Hyperledger Explorer](https://github.com/hyperledger/blockchain-explorer)" or perhaps "Hyperledger Fabric explorer" if you want to mention Fabric

> * Resolved an issue with some Hyperledger Fabric Explorers being out of sync with the network state.

Author: ```suggestion
* Resolved an issue with some Hyperledger Explorer instances in Fabric networks being out of sync with the blockchain state.
```
